.storybook/config.js

```diff
@@ -2,18 +2,24 @@ import { withInfo } from '@storybook/addon-info';
 import { withOptions } from '@storybook/addon-options';
 import { addDecorator, configure } from '@storybook/react';
 
-import * as themes from '../src/storybook-addon/themes';
-import { withThemes } from '../src/storybook-addon/withThemes';
+import * as themes from './custom-addons/withThemes//themes';
```

Comment:
I think we could move this to a separate package since it's used in every single project that requires UI-Kit.
I will do it later.